.github/workflows/peribolos.yaml

```diff
@@ -0,0 +1,20 @@
+name: Apply Organization Membership
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - 'main'
+    paths:
+      - "config/opendatahub-io/org.yaml"
+  pull_request_review:
```

Comment:
I assume that you only want this to run when `main` is updated so this line is not needed since it breaks the yaml

Reply:
Fixed.